Automattic/jetpack
# Jetpack: Unconnected site dashboard widget

#### Changes proposed in this Pull Request:

* Adds wp-admin dashboard widget for unconnected sites according to the changes proposed in pdpAdu-FT-p2

#### Does this pull request change what data or activity we track or use?

This PR does not add data tracking.

#### Other information:


#### Jetpack product discussion


pbtFPC-34k-p2

#### Testing instructions:






> You should be able to connect to WordPress.com in order to test this feature

* Pull this branch and run on your local environment.
* You might need to `jetpack docker uninstall && jetpack docker install` if you had a local running version.
* When logging in as admin, you should see the connection banner, dismiss it and refresh the page. You should get the connection widget when visiting the admin dashboard after dismissing the Jetpack connection banner.
* You should only see the Jetpack connection widget for unconnected sites.
* You should be able to click the link on the widget to start connecting your site.


#### Screenshots
![image](https://user-images.githubusercontent.com/5550190/195120862-091c440d-3a58-4b5e-988f-a45181e75512.png)


**Current look with the addition of Terms of Service disclaimer**
![image](https://user-images.githubusercontent.com/5550190/197513234-26af2cd1-a9cc-44de-a231-952071a24144.png)
